### ISSUE TITLE
notes for you!

### ISSUE THREAD

**Issue by department19** [fac30/agency-site--oleg-elena#39]
- consider using `:focus` pseudo-selector to replicate the hover animation in revealing names when using keyboard navigation
- screen reader cannot read the flavour text of team members
- contact form max length attribute should be considered
